File: Ruby/1037.rb
num = gets.chomp.to_f

if num > 75 and num <= 100
  puts "Intervalo (75,100]"

elsif num > 50 and num <= 75
  puts "Intervalo (50,75]"

elsif num > 25 and num <= 50
  puts "Intervalo (25,50]"

elsif num >= 0 and num <= 25
  puts "Intervalo [0,25]"

else 
  puts "Fora de intervalo"

end

File: C++/1037.cpp
#include <iostream>

using namespace std;

int main() {
    float num;
    
    cin >> num;
    
    if(num < 0.0000)
         cout << "Fora de intervalo\n";

    else if(num >= 0.0000 && num <= 25.0000)
         cout << "Intervalo [0,25]\n";

    else if(num > 25.0000 && num <=50.0000)
         cout << "Intervalo (25,50]\n";

    else if(num > 50.0000 && num <=75.0000)
         cout << "Intervalo (50,75]\n";

    else if(num > 75.0000 && num <=100.0000)
         cout << "Intervalo (75,100]\n";

    else if(num > 100.0000)
         cout << "Fora de intervalo\n";
         
    return 0;
}